### PR TITLE
refactor(tests): Tests to Fixtures - simplify plugin

### PR DESCRIPTION
**Pull request by sylvesteraswin**
Changed all `thePlugins` files to `actual` and `expected` files.

### CODE REVIEW

File: packages/babel-plugin-transform-regexp-constructors/__tests__/fixtures/escape/actual.js [hunk 1]
```diff
@@ -0,0 +1 @@
+String.raw`var x = new RegExp('\\w+\\s')`;
```
Comment (boopathi) [inline packages/babel-plugin-transform-regexp-constructors/__tests__/fixtures/escape/actual.js]: remove string.raw

Comment (vigneshshanmugam) [reply packages/babel-plugin-transform-regexp-constructors/__tests__/fixtures/escape/actual.js]: It was rebase screw up.

Comment (boopathi) [reply packages/babel-plugin-transform-regexp-constructors/__tests__/fixtures/escape/actual.js]: but why is regex constructors in your pr ?

**Comment by vigneshshanmugam** [comment]
@sylvesteraswin Ji, rebase it from latest master.

File: packages/babel-plugin-minify-simplify/__tests__/fixtures/bail-on-different-member-exprs/actual.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+if (a) {
+  this.a = 1;
+} else {
+  this.b = 2;
+}
```
Comment (boopathi) [inline packages/babel-plugin-minify-simplify/__tests__/fixtures/bail-on-different-member-exprs/actual.js]: the filename looks wrong. It doesn't bail here. `if-else-member-to-conditional` ?

File: packages/babel-plugin-minify-simplify/__tests__/fixtures/bail-on-different-operator/actual.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+if (a) {
+  x.b += foo;
+} else if (b) {
+  x.b -= bar;
+} else {
+  x.b += baz;
+}
```
Comment (boopathi) [inline packages/babel-plugin-minify-simplify/__tests__/fixtures/bail-on-different-operator/actual.js]: filename: `if-else-member-to-conditional-2` ?

File: packages/babel-plugin-minify-simplify/__tests__/fixtures/bail-rearranging-conditionals-for/actual.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+var x;
+if (a) {
+  x = foo;
+} else if (b) {
+  x = bar;
+} else {
+  baz();
+}
```
Comment (boopathi) [inline packages/babel-plugin-minify-simplify/__tests__/fixtures/bail-rearranging-conditionals-for/actual.js]: filename: `if-else-to-conditional`

File: packages/babel-plugin-minify-simplify/__tests__/fixtures/combinie-returns/actual.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+    if (a.b) {
+      if(a.b.c) {
+        if(a.b.c()){
+          return;
+        }
+      }
+    }
+  }
+  for (; true;) wat();
+}
```
Comment (boopathi) [inline packages/babel-plugin-minify-simplify/__tests__/fixtures/combinie-returns/actual.js]: filename: typo `s/combinie/combine`

File: packages/babel-plugin-minify-simplify/__tests__/fixtures/conditionals-for/actual.js [hunk 6]
```diff
@@ -0,0 +1,8 @@
+var x;
+if (a) {
+  x = foo;
+} else if (b) {
+  x = bar;
+} else {
+  x = baz;
+}
```
Comment (boopathi) [inline packages/babel-plugin-minify-simplify/__tests__/fixtures/conditionals-for/actual.js]: filename: `if-else-to-conditional-2` ?

File: packages/babel-plugin-minify-simplify/__tests__/fixtures/consider-hoisted-definitions-if-return/actual.js [hunk 7]
```diff
@@ -0,0 +1,9 @@
+function foo() {
+  bar();
+  if(x) return;
+  const {a}=b;
+  function bar () {
+    baz();
+    bar();
+  }
+}
```
Comment (boopathi) [inline packages/babel-plugin-minify-simplify/__tests__/fixtures/consider-hoisted-definitions-if-return/actual.js]: filename: `if-return`

File: packages/babel-plugin-minify-simplify/__tests__/fixtures/convert-consequents-switch-into-sequence-expressions/actual.js [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+    case 'foo':
+      bar();
+      foo();
+      break;
+    case 'bar':
+      wow();
+      return wo;
+      break;
+  }
+}
```
Comment (boopathi) [inline packages/babel-plugin-minify-simplify/__tests__/fixtures/convert-consequents-switch-into-sequence-expressions/actual.js]: filename: `switch-consequents-to-seq`

File: packages/babel-plugin-minify-simplify/__tests__/fixtures/convert-guarded-not-ors/actual.js [hunk 9]
```diff
@@ -0,0 +1,2 @@
+x();
+if (!foo.bar) foo.bar = wat;
```
Comment (boopathi) [inline packages/babel-plugin-minify-simplify/__tests__/fixtures/convert-guarded-not-ors/actual.js]: filename: `to-guarded-expr`

File: packages/babel-plugin-minify-simplify/__tests__/fixtures/convert-guarded-nots-ors/actual.js [hunk 10]
```diff
@@ -0,0 +1,4 @@
+if (!foo && foo !== bar) {
+  wow();
+  such();
+}
```
Comment (boopathi) [inline packages/babel-plugin-minify-simplify/__tests__/fixtures/convert-guarded-nots-ors/actual.js]: filename: `to-guarded-expr-2`

**Comment by boopathi** [comment]
will be superseded by a different pr by @sylvesteraswin

**Pull request closed without merging**